### ISSUE TITLE
Change death sound of support commanders to not be global anymore

### ISSUE THREAD

**Issue by Kami-Interface** [LOUD-Community-Edition/LOUD-Community-Edition#134]

**Comment by Azraeel** [comment]
@Kami-Interface think u can handle this one?

**Comment by Kami-Interface** [comment]
I'll give it a go once I'm back next week

**Comment by PurpleMossClump** [comment]
Killed      = Sound { Bank = 'Explosions', Cue = 'Aeon_Nuke_Impact', LodCutoff = 'UnitMove_LodCutoff'},

Is the sound global because it's using 'Aeon_Nuke_Impact', which I'm assuming is a global sound? 
If so, one would just need to change that sound to something else, yeah?

**Comment by Kami-Interface** [comment]
Yup exactly, they're using the nuke sound instead of their original sacu sound

**Comment by PurpleMossClump** [comment]
Do you know where/how to find the original sound?

**Comment by Kami-Interface** [comment]
I'd imagine it's available when looking into the folder for the vanilla units since the game can still be launched vanilla, without any of the mods

**Comment by PurpleMossClump** [comment]
I'm looking side-by-side between the LOUD unit BP and the Vanilla BP and the parameters are the same.

Vanilla
Killed = Sound {
            Bank = 'Explosions',
            Cue = 'Aeon_Nuke_Impact',
            LodCutoff = 'UnitMove_LodCutoff',
        },
        
LOUD
Killed      = Sound { Bank = 'Explosions', Cue = 'Aeon_Nuke_Impact', LodCutoff = 'UnitMove_LodCutoff'},

**Comment by Kami-Interface** [comment]
No way that the sacu death sound is global in vanilla as well

**Comment by Kami-Interface** [comment]
I'm not home this weekend, but if you want a test it further you could check the game by launching directly from steam and disabling the mods, and then see if sacu has global death sound

**Comment by PurpleMossClump** [comment]
I just tested it in Vanilla. It is the same sound, but the sound is not global.
